rhizome/host/lib/vm_setup.rb

```diff
@@ -364,6 +364,12 @@ def build_nftables_config(gua, nics, ip4)
       }
       # NAT4 rules
       #{generate_nat4_rules(ip4, nics.first.net4)}
+      table inet fw_table {
+        chain forward_ingress {
+          type filter hook forward priority filter; policy drop;
+          ip saddr 0.0.0.0/0 tcp dport 22 ip daddr #{nics.first.net4} counter accept
```

Comment:
port 22 is always open? would this be closed after sshable is confirmed?

Reply:
It is removed immediately after we switch to `wait` state because we apply firewall rules, there.